app/src/main/java/com/honeybeard/androidfundamentals/activities/movie_details/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,51 @@
+package com.honeybeard.androidfundamentals.activities.movie_details
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.honeybeard.androidfundamentals.R
+
+// TODO: Rename parameter arguments, choose names that match
+// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+private const val ARG_PARAM1 = "param1"
+private const val ARG_PARAM2 = "param2"
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [MovieDetailsFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class MovieDetailsFragment : Fragment() {
```

Comment:
**OPTIONAL**
you can use instead of override `onCreateView `
like so `class MovieDetailsFragment : Fragment(R.layout.fragment_movie_details)`